src/coreclr/src/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -353,7 +353,7 @@ public static long GetTotalMemory(bool forceFullCollection)
         }
 
         [DllImport(RuntimeHelpers.QCall, CharSet = CharSet.Unicode)]
-        private static extern IntPtr _RegisterFrozenSegment(IntPtr sectionAddress, IntPtr sectionSize);
+        private static extern IntPtr _RegisterFrozenSegment(IntPtr sectionAddress, nint sectionSize);
```

Comment:
The unmanaged size is using unsigned type for this, but fixing that would require changes in the code that looks up this API via private reflection.

Reply:
Right - the unmanaged code is using `SIZE_T`.

https://github.com/dotnet/runtime/blob/ab49e0f9dcd56958148827c6b47428b56187b5f8/src/coreclr/src/vm/comutilnative.cpp#L1127-L1133

I didn't want to actually change the type. Unless @mjsabby was OK with it.  I assume that is the only user of this experimental feature, or are there others?

Reply:
Oops, I merged this -- perhaps there was an open question here?